publish/dir.targets

```diff
@@ -9,7 +9,7 @@
     <ItemGroup>
       <PublishFile Include="@(CompressedFile)" />
       <PublishFile Include="@(RuntimePackageFile)" />
-      <PublishFile Condition="'$(PublishRidAgnosticPackages)' == 'true'" Include="@(IdentityPackageFile)" />
+      <PublishFile Condition="'$(PublishRidAgnosticPackages)' == 'true'" Include="@(RidAgnosticPackageFile)" />
```

Comment:
I did this on my local machine:

```
/home/matell/git/core-setup/Tools/msbuild.sh /home/matell/git/core-setup/publish/publish.proj /t:GatherPublishToAzureBinaries /flp:v=diag /p:DistroRid=ubuntu.14.04-x64 /p:TargetArchitecture=x64 /p:ConfigurationGroup=Debug /p:PortableBuild=false /p:OSGroup=Linux
```

And ensured that the stuff added to the itemgroups to publish did not contain the new packages:

```
Target "GatherPublishToAzureBinaries: (TargetId:4)" in file "/home/matell/git/core-setup/publish/dir.targets" from project "/home/matell/git/core-setup/publish/publish.proj" (entry point):
Set Property: RelativeBlobPath=master/Binaries/2.0.0-preview2-25311-9
Added Item(s):
    PublishFile=
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/dotnet-hostfxr-ubuntu.14.04-x64.2.0.0-preview2-25311-9.tar.gz
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/dotnet-sharedframework-symbols-ubuntu.14.04-x64.2.0.0-preview2-25311-9.tar.gz
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/dotnet-sharedframework-ubuntu.14.04-x64.2.0.0-preview2-25311-9.tar.gz
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/dotnet-ubuntu.14.04-x64.2.0.0-preview2-25311-9.tar.gz
Added Item(s):
    PublishFile=
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.App.2.0.0-preview2-25311-9.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.DotNetAppHost.2.0.0-preview2-25311-9.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.DotNetHost.2.0.0-preview2-25311-9.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.DotNetHostPolicy.2.0.0-preview2-25311-9.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.DotNetHostResolver.2.0.0-preview2-25311-9.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.App.2.0.0-preview2-25311-9.symbols.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.DotNetAppHost.2.0.0-preview2-25311-9.symbols.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.DotNetHost.2.0.0-preview2-25311-9.symbols.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.DotNetHostPolicy.2.0.0-preview2-25311-9.symbols.nupkg
        /home/matell/git/core-setup/Bin/ubuntu.14.04-x64.Debug/packages/runtime.ubuntu.14.04-x64.Microsoft.NETCore.DotNetHostResolver.2.0.0-preview2-25311-9.symbols.nupkg
```

So these should not be uploaded to blob storage when running on nix.